Zend/zend_generators.stub.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+final class Generator implements Iterator
+{
+    /** @return void */
```

Comment:
As this is a final class, it should be fine to use `: void`. (Same below.)

Reply:
Thanks!